conjure-java-core/src/integrationInput/java/test/prefix/com/palantir/product/OptionalAlias.java

```diff
@@ -25,7 +25,6 @@ private OptionalAlias() {
     }
 
     @JsonValue
-    @Safe
     public Optional<@Safe String> get() {
```

Comment:
No need to annotate both the method and return value, this deduplicates our safety annotations to the most specific one.